src/package.xml

```diff
@@ -932,6 +965,7 @@
     <types>
         <members>Attribute__c.Credential</members>
         <members>Attribute__c.Student_Characteristic</members>
+        <members>Case.Incident</members>
```

Comment:
I don't know exactly what our process is on introducing record types on standard objects. Similar to account, does this maybe need to be unmanaged? Not sure if all of our existing customers should get this by default or not, if they're already heavily using the Case object and don't plan on using Behavior Management...

Reply:
I was going to recommend we move these to unmanaged as well for that reason. I was reviewing the spec and didn't see where we specified one way or the other. I think perhaps we want to go with unmanaged here if possible. That may mean that the page layouts associated will likely have to be unmanaged as well. (worth considering)

Reply:
On the fence about the page layout being unmanaged. I think it can be managed like the account page layouts (consistency?), but I'd like to know how it gets assigned to the correct record type. 

It looks like we assign page layouts to the admin.profile in unpackaged/config/dev/profiles/Admin.profile (and perhaps elsewhere?) so we might want to do something similar here.

Reply:
And for existing customers they'd have to assign it themselves, presumably...

Reply:
yea you're correct. I'm stuck in K12 land at the moment where we can't do stuff with page layouts because the fields won't be there. Not the case (no pun intended) here :)

Reply:
Trying to clarify here, we do want to keep case record type and page layout as managed and change the business process to unmanaged?

Reply:
The Case record type should be unmanaged, but the page layout can remain in the package? I think?

Reply:
correct and I apologize if I've steered you wrong somewhere. Balancing the odd unmanaged/managed K12 stuff and the EDA stuff is messing with my head. For this feature (which is on EDA) Record Types should be unmanaged and page layouts can be managed.